packages/accounts-base/accounts_tests.js

```diff
@@ -623,3 +623,44 @@ Tinytest.add(
     Accounts._options = accountsOptions;
   }
 );
+
+Tinytest.add(
+    'accounts - verify beforeExternalLogin hook can stop user login',
```

Comment:
seems like this is not documented or added to the change notes

Reply:
@hexsprite this is documented here https://github.com/meteor/docs/pull/677

Reply:
thank you. seems like all it needs a references in CHANGES then